File: aula06/1216.py
# 1216

D, n = 0, 0

while True:
  try:
    nome = input()
    distancia = float(input())
    
    D += distancia
    n += 1
  except:
    break

if n > 0:
  print(f"{(D / n):.1f}")

File: aula10/1216.c
#include <stdio.h>

int main() {
  double total = 0, distancia;
  int i = 0;
  char nome[50];
  
  while (1) {
    if (fgets(nome, 50, stdin) == NULL){
      break;
    }         

    scanf("%lf", &distancia);
    getchar();
    
    total += distancia;
    i++;
  }
  
  printf("%.1lf\n", total / i);

  return 0;
}